app/bundles/CoreBundle/Test/Templating/Helper/TranslatorHelperTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Mautic\CoreBundle\Test\Templating\Helper;
+
+use Mautic\CoreBundle\Templating\Helper\TranslatorHelper;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Translation\Translator;
+
+class TranslatorHelperTest extends TestCase
+{
+    public function testGetJsLang(): void
+    {
+        $translator = $this->createMock(Translator::class);
+        $translator->expects($this->exactly(2))->method('getCatalogue')->willReturnCallback(
+            function () use ($translator) {
+                return new TranslatorCatalogue($translator);
+            }
+        );
+
+        $translatorHelper = new TranslatorHelper($translator);
+        $translatorHelper->getJsLang();
```

Comment:
Shouldn't the output of this method be asserted to some value? I'm not sure what this test is testing otherwise?

Reply:
Do not need. Tests bellow cover it too